CMakeLists.txt

```diff
@@ -99,6 +99,11 @@ endif()
 
 include(CheckLibraryExists)
 check_library_exists(execinfo backtrace "" HAVE_LIBEXECINFO)
+check_include_file("execinfo.h" EXECINFOH)
```

Comment:
Can you move the new code above `check_library_exists` block i.e., just above `include`? This one is dedicated to adding `-lexecinfo` on BSDs or nothing on Linux/glibc.